### PR TITLE
chore(ci): Fix lerna concurrency

### PR DESCRIPTION
**Pull request by sidharthachatterjee**
The new dependency updates seem to cause https://github.com/lerna/lerna/issues/1798 and `bootstrap` fails as a result on CircleCI. 

This is happening because CircleCI incorrectly reports the number of CPUs and lerna tries to run too many at once and runs out of memory. 

The fix here is to set concurrency to 2 for `lerna prepare` (only when running in CI)

### CODE REVIEW

**Comment by sidharthachatterjee** [comment]
@KyleAMathews Yeah, really frustrating that they misreport `os.cpus()`

**Comment by wardpeet** [comment]
Please think of windows users :) This change won't allow me to bootstrap. 

`\"lerna-prepare -- --{@}\" --"` this voodoo is breaking windows.
`npm-run-all -s  "lerna-prepare -- --{@}" -- --concurrency=2`

**Comment by sidharthachatterjee** [comment]
Oops! Sorry, totally didn't think of that @wardpeet 😱

File: package.json [hunk 1]
```diff
@@ -72,7 +72,7 @@
     }
   },
   "scripts": {
-    "bootstrap": "npm-run-all -s check-versions lerna-prepare",
+    "bootstrap": "npm-run-all -s check-versions \"lerna-prepare -- --{@}\" --",
```
Comment (wardpeet) [inline package.json]: We use yarn on CI so we shouldn't need lerna-prepare with -- as yarn works without -- to pass arguments down.
```suggestion
    "bootstrap": "npm-run-all -s check-versions lerna-prepare",
```

Comment (sidharthachatterjee) [reply package.json]: Already tried this. It doesn't work because both `check-versions` and `lerna-prepare` are scripts in `package.json`

**Comment by sidharthachatterjee** [comment]
@wardpeet Voodoo seemed to have worked fine on Windows ;)

https://circleci.com/gh/gatsbyjs/gatsby/203683?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

This runs `yarn bootstrap`

File: package.json (hunk 1 above)
Comment (wardpeet) [reply package.json]: My bad, i had a dependency error, not a lerna-prepare issue. This works on windows too! :) great job! Sorry about being faulty on this one! <3

**Comment by sidharthachatterjee** [comment]
This was caused by https://github.com/lerna/lerna/pull/1931 by the way which was in a new `lerna` version which we upgraded to recently in our mass upgrade!

**Pull request merged**